scripts/uberenv/uberenv.py

```diff
@@ -224,12 +232,6 @@ def is_darwin():
 def is_windows():
     return "windows" in platform.system().lower()
 
-def is_wsl():
```

Comment:
@jamiebramwell is this still needed for WSL/WSL2 support?

Reply:
I don't think so.